snews_pt/snews_pub.py

```diff
@@ -59,9 +60,15 @@ def send(self, messages):
         if type(messages) == dict:
             messages = list(messages)
         for message in messages:
-            self.stream.write(message)
-            self.display_message(message)
-
+            if snews_pt_utils.is_snews_format(message):
+                self.stream.write(message)
+                self.display_message(message)
+            else:
+                click.secho(f'{"-" * 64}', fg='bright_red')
+                click.secho(f'Skipping message! Improper format! See log for details.')
```

Comment:
Since it is only when checking the user input, isn't it easier to raise a warning rather than logging everything and directing user to the logs?

Reply:
Yeah that sounds like a better way to do it.  I just did it this way because I already knew how to do logs and not warnings :wink: